### PR TITLE
style: FilterSelect style tweaks

### PR DESCRIPTION
**Pull request by mistercrunch**
## After
Screen Shot 2020-07-27 at 8 33 38 PM

## Before
Screen Shot 2020-07-27 at 8 35 18 PM

### CODE REVIEW

File: superset-frontend/src/components/ListView/Filters.tsx [hunk 1]
```diff
@@ -48,9 +48,12 @@ interface SelectFilterProps extends BaseFilter {
   paginate?: boolean;
 }
 
+const FONT_SIZE = 12;
+
 const FilterContainer = styled.div`
   display: inline-flex;
   margin-right: 2em;
+  font-size: ${FONT_SIZE}px;
```
Comment (rusackas) [inline superset-frontend/src/components/ListView/Filters.tsx:56]:
```suggestion
  font-size: ${({ theme }) => theme.typography.sizes.s};
```

File: superset-frontend/src/components/ListView/Filters.tsx [hunk 2]
```diff
@@ -63,6 +66,7 @@ const filterSelectTheme: PartialThemeConfig = {
   spacing: {
     baseUnit: 2,
     minWidth: '5em',
+    fontSize: FONT_SIZE,
```
Comment (rusackas) [inline superset-frontend/src/components/ListView/Filters.tsx:69]:
```suggestion
    fontSize: ${({ theme }) => theme.typography.sizes.s},
```

File: superset-frontend/src/components/ListView/Filters.tsx [hunk 3]
```diff
@@ -48,9 +48,12 @@ interface SelectFilterProps extends BaseFilter {
   paginate?: boolean;
 }
 
+const FONT_SIZE = 12;
```
Comment (rusackas) [inline superset-frontend/src/components/ListView/Filters.tsx:51]:
```suggestion
```

File: superset-frontend/src/components/ListView/Filters.tsx (hunk 2 above)
Comment (mistercrunch) [reply superset-frontend/src/components/ListView/Filters.tsx:69]: wait I don't think that works, I need to outright import the theme I think

**Comment by rusackas** [comment]
Added a couple suggestions using the theme variables instead of the hard-coded 12px. Otherwise, simple enough :) 👍

File: superset-frontend/src/components/ListView/Filters.tsx (hunk 2 above)
Comment (rusackas) [reply superset-frontend/src/components/ListView/Filters.tsx]: Nope! It should be available via the ThemeProvider way up at the top of the app.

Comment (rusackas) [reply superset-frontend/src/components/ListView/Filters.tsx]: In fact, it's _better_ to use the theme through the provider than importing it from the `superset-ui` package. Then we can do fun stuff with it, e.g. `<ThemeProvider theme={...supersetTheme, ...darkMode, ...myCompanyBrandColors}>`

Comment (mistercrunch) [reply superset-frontend/src/components/ListView/Filters.tsx]: wait I'm not in a text string here, this is a good old JS object... I think we'd have to convert those objects to emotion first...

**Comment by rusackas** [comment]
Added some suggestions in a branch off of your branch., and opened a PR on the fork for your consideration.

File: superset-frontend/src/components/ListView/Filters.tsx (hunk 2 above)
Comment (rusackas) [reply superset-frontend/src/components/ListView/Filters.tsx]: Just co comment... addressed this in subsequent commits.

**Comment by rusackas** [comment]
_shakes fist at Cypress CI process_

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
Impacts #8976

**Pull request merged**